x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/audit/logfile/LoggingAuditTrail.java

```diff
@@ -151,6 +158,8 @@
             "indices",
             (key) -> Setting.listSetting(key, Collections.singletonList("*"), Function.identity(), Property.NodeScope, Property.Dynamic));
 
+    private static final Marker AUDIT_MARKER = MarkerManager.getMarker("AUDIT");
```

Comment:
Can we prefix this with `ES_`? I hope none of our dependencies are using `Marker` objects, but if they are we want to be extra sure that we don't get name conflicts.

Reply:
Good point! I have renamed the marker to `org.elasticsearch.xpack.security.audit` .

Please take another look.